internal/pkg/task/env_runner.go

```diff
@@ -39,9 +34,9 @@ type EnvRunner struct {
 	Env string
 
 	// Interfaces to interact with dependencies. Must not be nil.
-	VPCGetter     VPCGetter
-	ClusterGetter ResourceGetter
-	Starter       Runner
+	VPCGetter     vpcGetter
+	ClusterGetter clusterGetter
+	Starter       runner
```

Comment:
Why did we make the interfaces private if the fields are public? Wouldn't that make it harder for the clients to know what should be passed as input for these fields